PyOpenWorm/contextualize.py

```diff
@@ -208,14 +208,7 @@ def __setattr__(self, name, value):
             object.__setattr__(self, name, value)
         elif name == '__wrapped__':
             object.__setattr__(self, name, value)
-            try:
-                object.__delattr__(self, '__qualname__')
-            except AttributeError:
-                pass
-            try:
-                object.__setattr__(self, '__qualname__', value.__qualname__)
-            except AttributeError:
-                pass
+            object.__setattr__(self, '__qualname__', value.__qualname__)
```

Comment:
Now that I'm looking at this code again, I think this whole case can just be replaced by raising AttributeError -- setting `__wrapped__` isn't a legitimate use case for Contextualizables.

Reply:
Do you mean:
```
elif name == '__wrapped__':
  raise AttributeError
```
?

Reply:
Do you mean:
```
elif name == '__wrapped__':
  raise AttributeError
```
?